packages/experimental/RadioGroup/src/RadioGroup/radioGroupContext.ts

```diff
@@ -8,6 +8,7 @@ export interface RadioGroupContextValue extends RadioGroupState {}
 
 export const RadioGroupContext = React.createContext<RadioGroupContextValue>({
   value: null,
+  required: false,
```

Comment:
Why is this neeed in the context?

Reply:
In the web implementation, Radios listen and inherit the prop from RadioGroup. Stylistically and functionally, there's no change to Radio when required is true, but I'm guessing we still want to use it to report accessibilityState of Radios too.

Reply:
I agree, I'll remove it from Radio and the context.